python/katana_setup.py

```diff
@@ -138,17 +138,72 @@ def test_cython_module(name, cython_code, python_code="", extension_options=None
 
 
 def load_lang_config(lang):
+    """
+    Load the compilation configuration provided by CMake.
+
+    KatanaPythonSetupSubdirectory.cmake generates a JSON file that contains build and link flags that we read.
+    They typically look like:
+
+    .. code-block:: json
+
+        {
+        "COMPILER":"ccache;/home/amp/LocalInstalls/conda/miniconda3/envs/katana-dev/bin/clang++",
+        "INCLUDE_DIRECTORIES":"/home/amp/shared/Katana/katana-enterprise/libquery/include;/home/amp/shared/Katana/katana-enterprise/external/katana/libgalois/include;/home/amp/shared/Katana/katana-enterprise/external/katana/libtsuba/include;/home/amp/shared/Katana/katana-enterprise/external/katana/libsupport/include;/home/amp/shared/Katana/katana-enterprise/build-tests/external/katana/libsupport/include;/home/amp/LocalInstalls/conda/miniconda3/envs/katana-dev/include;/home/amp/LocalInstalls/conda/miniconda3/envs/katana-dev/lib/backward;/home/amp/shared/Katana/katana-enterprise/libgluon/include;/home/amp/shared/Katana/katana-enterprise/libdist/include;/home/amp/shared/Katana/katana-enterprise/libtsuba-fs/include;/home/amp/shared/Katana/katana-enterprise/external/katana/libtsuba/src",
```

Comment:
suggestion: feel free to elide bits and just keep the `LINKER` and `SHELL` portions:

```
.. code-block:: json
    {
     "COMPILER": ....
     "INCLUDE_DIRECTORIES": .....
     "LINK_OPTIONS": <as before>
     "LINKER_WRAPPER_FLAG": <as before>
     "LINKER_WRAPPER_FLAG_SEP": <as before>
    }
```